chapters/6-analysis.tex

```diff
@@ -42,62 +46,74 @@
+The derived indicators you will construct should be
+planned during research design\index{Research design};
+the pre-analysis plan should serve as your guide
+at this stage.\index{Pre-analysis plan}
+During construction, data will typically be
 reshaped, merged, and aggregated to change the level of the data points
-from the unit of \textit{observation} in the original data 
-to the unit of \textit{analysis} in the constructed data.\sidenote{\url{
+from the \textbf{unit of \textit{observation}} in the original data
+to the \textbf{unit of \textit{analysis}}.\sidenote{\url{
 		https://dimewiki.worldbank.org/Unit\_of\_Observation}}
 
 % A project may require multiple purpose-built data sets
 A constructed dataset is built to answer an analysis question.
-Since different pieces of analysis may require different sub-samples,
-or even different units of observation,
-you may have one or many constructed datasets,
-depending on your analysis plan.
-You often cannot create a single ``canonical'' analysis dataset.
-It is common to have many purpose-built analysis datasets.
+If the sub-samples and units of observation 
+vary for different pieces of the analysis
+you may need many constructed datasets.
+It is often not a good practice
+to create a single ``canonical'' analysis dataset.
+It is often better practice to have 
+many purpose-built analysis datasets.
 For a concrete example of what this means,
-think of an agricultural intervention that was randomized across villages
+think of an agricultural intervention 
+that was randomized across villages
 and only affected certain plots within each village.
-The research team may want to run household-level regressions on income,
+The research team may want to 
+run household-level regressions on income,
 test for plot-level productivity gains,
 and check if village characteristics are balanced.
 Having three separate datasets for each of these three pieces of analysis
-will result in much cleaner do-files than if they all started from 
-a single analysis dataset that constantly needs to be transformed.
+will result in cleaner and more efficient analytical code than if
+you started from a single analysis dataset and repeatedly transformed it.
 
 \subsection{Fitting construction into the data workflow}
 
 % Why construction is separate from data cleaning
-Construction is done after from data cleaning for two reasons.
-First, to clearly differentiate correction of data entry errors
-(necessary for all interactions with the data)
-from creation of analysis indicators (necessary only for specific analyses).
-Second, to ensure that variable definitions are consistent across.
+Construction follows data cleaning and 
+should be treated as a separate task for two reasons.
+First, this helps to clearly differentiate error corrections 
+(necessary for all data uses)
+from creation of analysis indicators 
+(necessary only for specific analyses).
+Second, it helps to ensures that variable definitions are
```

Comment:
I think this is more connected to analysis then cleaning? Not sure how it ended up in this paragraph (probably me getting confused)